### PR TITLE
[node, client, contracts] Upgrade cf

### PR DESCRIPTION
**Pull request by LayneHaber**
## The Problem
CF removed some of `ProposeVirtualInstall`

## The Solution
Updated everything to build

### CODE REVIEW

File: modules/types/package.json [hunk 1]
```diff
@@ -13,11 +13,11 @@
     "test": "./node_modules/.bin/jest"
   },
   "dependencies": {
+    "@connext/messaging": "0.1.2",
```
Comment (ArjunBhuptani) [inline modules/types/package.json]: Shouldn't this be updated?

Comment (ArjunBhuptani) [reply modules/types/package.json]: Also, why do we have messaging in our types?

Comment (LayneHaber) [reply modules/types/package.json]: ah merge conflict most likely

Comment (LayneHaber) [reply modules/types/package.json]: fixed

**Pull request merged**